service/controller/v2/arm_templates/vmss.json

```diff
@@ -70,6 +73,21 @@
 
         "roleAssignmentName": "[guid(concat(parameters('vmssName'), '-', 'roleassignment'))]",
 
+        "vmssExtensions": [
+            {
+                "name": "MSILinuxExtension",
+                "properties": {
+                    "publisher": "Microsoft.ManagedIdentity",
+                    "type": "ManagedIdentityExtensionForLinux",
+                    "typeHandlerVersion": "1.0",
+                    "autoUpgradeMinorVersion": true,
+                    "settings": {
+                        "port": 50342
+                    },
+                    "protectedSettings": {}
+                }
+            }
+        ],
```

Comment:
You also need to disable roleAssignments. You can use condition like here

https://github.com/giantswarm/azure-operator/blob/master/service/controller/v2/arm_templates/route_table_setup.json#L37

Reply:
It's done already: https://github.com/giantswarm/azure-operator/pull/245/files#diff-b64f09db2486367b61a55095d8016d16R278